packages/playground/windows/playground/HostingPane.xaml

```diff
@@ -141,6 +141,16 @@
         VerticalAlignment="Center"
         IsChecked="true"
         Content="Live Reload"/>
+      <TextBox x:Name="x_DebuggerPort"
+        Text="9229"
```

Comment:
>9229 [](start = 14, length = 4)

use a Binding instead of doing this

Something like Text="{Binding DebuggerPort, StringFormat={}{0}}"
(please test that this works two-ways)

Reply:
or even 
```
<Page 
     xmlns:sys="using:System"
     xmlns:local="using:MyNamespace">
     ...
     <CalendarDatePicker Date="{x:Bind sys:DateTime.Parse(TextBlock1.Text)}" />
     <TextBlock Text="{x:Bind sys:String.Format('{0} is now available in {1}', local:MyPage.personName, local:MyPage.location)}" />
</Page>
```

Reply:
My apologies, I am not super familiar with this part.
I updated the code but I am not sure if it is what you had in mind. Would you mind taking another look?

Reply:
sure - my point was about your current code is having to do the conversion between string and int manually, where instead I was suggesting that you use the x:Bind sys:String.Format syntax which does that for you, or if that isn't available in RS2, you could encapsulate the logic of the two-way conversion inside a ValueConverter like I mentioned above. See [this example](https://www.c-sharpcorner.com/article/xaml-value-converter-with-simple-example/) and the [docs](https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Data.IValueConverter)